templates/project/docs/.doctor-rst.yaml.twig

```diff
@@ -27,10 +27,12 @@ rules:
     no_bash_prompt: ~
     no_composer_phar: ~
     no_inheritdoc: ~
+    yaml_instead_of_yml_suffix: ~
+    no_app_bundle: ~
+    no_config_yaml: ~
+    kernel_instead_of_app_kernel: ~
+    no_app_console: ~
```

Comment:
This would bring up a lot of work 🤔

Reply:
There was `# can be enabled when no Symfony 3.4 support anymore` so I thought I could add this without issues.

Do you have any example ?

Reply:
Lets check it using a Proof PR:
https://github.com/sonata-project/SonataAdminBundle/pull/6357

Reply:
Wow it looks ok 🤔 

https://github.com/sonata-project/SonataAdminBundle/pull/6357/checks?check_run_id=1050311053

I will try another bundle

Reply:
Looks good:
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1106/checks?check_run_id=1050324214

https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1106

Lets merge it